basket/news/views.py

```diff
@@ -685,6 +686,7 @@ def lookup_user(request):
 
     token = request.GET.get('token', None)
     email = request.GET.get('email', None)
+    get_fxa = 'fxa' in request.GET
```

Comment:
Just to clarify, we don't care what the value of the `fxa` key is in the URL, only that the key is present, right?

Reply:
I think that's the case. Bedrock will be sending `?fxa=1` but really it's just acting as a boolean indicating a desire to have the fxa status returned. I don't think we need to support an off state like `?fxa=0` or anything like that.